data_override/include/data_override_impl.inc

```diff
@@ -191,6 +115,7 @@ subroutine data_override_init(Atm_domain_in, Ocean_domain_in, Ice_domain_in, Lan
   character(len=128)    :: grid_file = 'INPUT/grid_spec.nc'
   integer               :: is,ie,js,je,use_get_grid_version
   integer               :: i, unit, io_status, ierr
+  logical               :: atm_on, ocn_on, lnd_on, ice_on, lndUG_on
```

Comment:
These variables need documentation

Reply:
Documentation for `DATA_OVERRIDE_INIT_IMPL_` arguments added in 2478bf6cf73db4c1126b1b29a50edd75a1deefae.